certifier_service/certlib/certlib_proofs.go

```diff
@@ -25,8 +25,6 @@ import (
 	"crypto/x509"
 	"google.golang.org/protobuf/proto"
 	certprotos "github.com/jlmucb/crypto/v2/certifier-framework-for-confidential-computing/certifier_service/certprotos"
-	oeverify "github.com/jlmucb/crypto/v2/certifier-framework-for-confidential-computing/certifier_service/oeverify"
```

Comment:
If I have these, I run into these warnings, causing the build to fail:

```
sdb-fdb-build:[39] $ cd $CERTIFIER_PROTOTYPE/certifier_service

sdb-fdb-build:[40] $ go build simpleserver.go
# github.com/jlmucb/crypto/v2/certifier-framework-for-confidential-computing/certifier_service/certlib
certlib/certlib_proofs.go:28:2: imported and not used: "github.com/jlmucb/crypto/v2/certifier-framework-for-confidential-computing/certifier_service/oeverify"
certlib/certlib_proofs.go:29:2: imported and not used: "github.com/jlmucb/crypto/v2/certifier-framework-for-confidential-computing/certifier_service/gramineverify"

sdb-fdb-build:[41] $ echo $?
2
```

`git blame` on `/main` shows that these were recently uncommented & included. Not sure why the builds were still succeeding in CI. 

And, do we need these imports? The history is murky to me ... and will need to be resolved before this commit can land in `/main`. 

(I'm afraid that by deleting these lines I might be inadvertently regressing something else.)

Reply:
I was going to talk about this at the meting today.  Go does not have conditional compilation so these are needed for the OE and gramine certifiers when they are being used.  However, if you don't have them on your machine, you can't compile so I ususally comment them out when I'm testing (along with the implementing code in InitProvedStatements).  There must be a better way and I was going to ask for suggestions.